gensrc/proto/starlake.proto

```diff
@@ -57,4 +60,30 @@ message TxnLogPB {
     optional OpSchemaChange op_schema_change = 5;
 }
 
+message PublishVersionRequest {
+    repeated int64 tablet_ids = 1;
+    repeated int64 txn_ids = 2;
+    optional int64 base_version = 3;
```

Comment:
Why need base_version?

Reply:
When sending a publish version task for a schema change job, the base version and the published version cae be discontinuous.

Reply:
`base_version = publish_version - txn_ids.size()`

This must be certain.

Reply:
updated